### PR TITLE
cobol_indent : scanner improvements, to be used for cartography

### PR DESCRIPTION
**Pull request by lefessan**
Some improvements to return more information in case this scanner is used instead of the COBOL parser to retrieve faster information (but less exact)

### CODE REVIEW

File: import/merlin
Comment (nberth) [inline import/merlin:1]: You probably need to `git submodule update`.

Comment (lefessan) [reply import/merlin:1]: Chez moi, ça passe, du coup, je ne comprends pas vraiment pourquoi ça échoue sur le CI...

Comment (nberth) [reply import/merlin:1]: Commit b3de92cd4fb23 reverts commits from submodules. You just need to edit that commit and restore the changes to submodules.

Comment (lefessan) [reply import/merlin:1]: What I don't understand is why it works locally on my computer and not on the CI, it should fail on my computer too, no ?

**Review by nberth — APPROVED** [review]

**Pull request merged**